ChatApp/ChatView/MessageCell.swift

```diff
@@ -0,0 +1,101 @@
+        contentView.backgroundColor = .clear
+        
+        label.textColor = .black
+        label.font = UIFont.systemFont(ofSize: CGFloat(Constants.labelFontSize))
+        label.numberOfLines = .zero
+        label.lineBreakMode = .byWordWrapping
+        
+        contentView.addSubview(containerView)
+        contentView.addSubview(textDate)
+        containerView.addSubview(label)
+        //        conteinerView.addSubview(elipse)
+        
+        setupContainerViewConstant()
+        setupLabelConstant()
+        setupTextDateConstant()
+    }
+    
+    func setupContainerViewConstant() {
+        containerView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            containerView.topAnchor.constraint(equalTo: contentView.topAnchor),
+            containerView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            containerView.trailingAnchor.constraint(equalTo: label.trailingAnchor, constant: CGFloat(Constants.containerViewTrailingAnchor))
+        ])
+    }
+    
+    func setupLabelConstant() {
+        label.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            label.topAnchor.constraint(equalTo: containerView.topAnchor, constant: CGFloat(Constants.labelTopAnchor)),
+            label.leadingAnchor.constraint(equalTo: containerView.leadingAnchor, constant: CGFloat(Constants.labelLeadingAnchor)),
+            label.trailingAnchor.constraint(equalTo: containerView.trailingAnchor, constant: CGFloat(Constants.labelTrailingAnchor)),
+            label.bottomAnchor.constraint(equalTo: containerView.bottomAnchor, constant: CGFloat(Constants.labelBottomAnchor))
+        ])
+    }
+    
+    func setupTextDateConstant() {
+        textDate.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            textDate.topAnchor.constraint(equalTo: containerView.bottomAnchor, constant: CGFloat(Constants.textDateTopAnchor)),
+            textDate.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: CGFloat(Constants.textDateLeadingAnchor)),
+            textDate.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: CGFloat(Constants.textDateBottomAnchor))
+        ])
+    }
+}
+
+extension MessageCell {
+    enum Constants {
+        static let containerViewTrailingAnchor = 16
+        static let textDateFontSize = 12
+        static let containerViewRadius = 25
+        static let labelFontSize = 16
+        static let labelTopAnchor = 16
+        static let labelLeadingAnchor = 16
+        static let labelTrailingAnchor = -16
+        static let labelBottomAnchor = -16
+        static let textDateTopAnchor = 4
+        static let textDateLeadingAnchor = 36
+        static let textDateBottomAnchor = -16
+        static let containerViewBackgroundColor = UIColor(hex: "F1F1F1")
+        static let textDateTextColor = UIColor(hex: "C7C7C7")
+    }
```

Comment:
separating and grouping Constraints is also good way that you can use in order to have more orginized code. 
```suggestion
    enum Constants {
        
        enum CollectionView {
            static let spacing = 16
            static let borderWidth = 1
            static let cornerRadius = 28
        }
        
        enum textView {
            static let leadingAnchor = 16
            static let height = 44
            static let textColor = UIColor(hex: "C7C7C7")
                ..........
    }
```